src/main/java/com/aws/greengrass/clientdevices/auth/certificate/CISShadowMonitor.java

```diff
@@ -273,11 +281,10 @@ private synchronized void processCISShadow(int version, Map<String, Object> desi
      * @param version CIS shadow version
      * @param desired CIS shadow <b>desired</b> state
      */
-    private void updateCISShadowReportedState(int version, Map<String, Object> desired) {
+    private void updateCISShadowReportedState(String version, Map<String, Object> desired) {
         LOGGER.atInfo().kv(VERSION, version).log("Reporting CIS version");
         UpdateShadowRequest updateShadowRequest = new UpdateShadowRequest();
         updateShadowRequest.thingName = shadowName;
-        updateShadowRequest.version = version;
```

Comment:
didn't we fix a bug where this request was failing because we didn't include a version? And now we're putting that bug back

Reply:
https://github.com/aws-greengrass/aws-greengrass-client-device-auth/pull/72

Perhaps version isn't needed.

Reply:
that was cause of missing `state` field, missing version should be fine, @jbutler just did a successful manual test